src/main/resources/db/migration/V1.0.0__create_songs_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS songs(
+ id INTEGER NOT NULL GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+ artistName VARCHAR(255) NOT NULL,
+ trackName VARCHAR(255) NOT NULL,
+ trackTime TIME NOT NULL,
+ genre VARCHAR (255) NOT NULL,
+ decade INTEGER NOT NULL,
+ emotion VARCHAR(255) NOT NULL
```

Comment:
Myślę, że tu trochę podobnie jak z gatunkami - emocje też powinny być jakimś osobnym zamkniętym zbiorem i raczej trzeba założyć, że to będzie coś charakterystycznego dla konkretnego użytkownika (zatem emocja byłaby cechą innej encji, na styku użytkownika i piosenki).

Podobnie jak gatunki, emocje sugerowałbym dodać później.